frontend/src/metabase/entities/collections.js

```diff
@@ -120,7 +119,7 @@ const Collections = createEntity({
             return canonicalCollectionId(collectionId);
           }
         }
-        return null;
+        return canonicalCollectionId(ROOT_COLLECTION.id);
```

Comment:
`canonicalCollectionId(ROOT_COLLECTION.id)` would return `null`, so this line shouldn't change any functionality. The selector fallbacks to root collection if even the personal collection is unavailable for some reason. This change just makes it explicit